serving/src/app/callback/store.rs

```diff
@@ -31,4 +31,5 @@ pub(crate) trait LocalStore {
         id: &str,
     ) -> Result<Vec<Arc<CallbackRequest>>, crate::Error>;
     async fn retrieve_datum(&mut self, id: &str) -> Result<Vec<Vec<u8>>, crate::Error>;
+    async fn is_available(&mut self) -> bool;
```

Comment:
```suggestion
    async fn ready(&mut self) -> bool;
```

`is` and `mut` doesn't sound right :)